exonum-java-binding/core/rust/src/handle/mod.rs

```diff
@@ -68,13 +39,6 @@ pub fn to_handle<T: 'static>(val: T) -> Handle {
     handle
 }
 
-/// Returns a handle (a raw pointer) to the given native-owned object. This handle should not be
-/// freed manually.
-pub fn as_handle<T>(val: &mut T) -> NonOwnedHandle<T> {
```

Comment:
I remember it was used in Node ITs — is that correct that it is not needed for newer, yet to be implemented tests?

Reply:
Not needed, as NodeContext is no more native-owned structure, but rather java-owned.

Reply:
Yeah, Node aspires to become one (ECR-2334) 😄